docs/user/security/authentication/index.asciidoc

```diff
@@ -85,45 +122,53 @@ Note that with `server.ssl.clientAuthentication` set to `required`, users are as
+  saml.saml1:
+    order: 0
+    realm: saml1
 --------------------------------------------------------------------------------
 
-. {kib} needs to specify which SAML realm in {es} should be used:
-+
-[source,yaml]
---------------------------------------------------------------------------------
-xpack.security.authc.saml.realm: realm-name
---------------------------------------------------------------------------------
+Users will be able to log in to {kib} via SAML Single Sign-On by navigating directly to the {kib} URL. Users who aren't authenticated are redirected to the Identity Provider for login. Most Identity Providers maintain a long-lived session—users who logged in to a different application using the same Identity Provider in the same browser are automatically authenticated. An exception is if {es} or the Identity Provider is configured to force user to re-authenticate. This login scenario is called _Service Provider initiated login_.
+
+It's also possible to configure multiple SAML authentication providers at the same time. In this case users will need to choose which provider to use for login at the Login Selector screen:
 
-. The Identify Provider sends authentication requests to the `Assertion Consumer Service` endpoint that {kib} exposes through a "non-safe" `POST` HTTP method. This does not include CSRF protection HTTP headers specific to {kib}. You must disable the CSRF check for this endpoint.
-+
 [source,yaml]
 --------------------------------------------------------------------------------
-server.xsrf.whitelist: [/api/security/saml/callback]
+xpack.security.authc.providers:
+  saml.saml1:
+    order: 0
+    realm: saml1
+    description: "Log in with Elastic"
+  saml.saml2:
+    order: 1
+    realm: saml2
+    description: "Log in with Auth0"
 --------------------------------------------------------------------------------
 
-Users will be able to log in to {kib} via SAML Single Sign-On by navigating directly to the {kib} URL. Users who aren't authenticated are redirected to the Identity Provider for login. Most Identity Providers maintain a long-lived session—users who logged in to a different application using the same Identity Provider in the same browser are automatically authenticated. An exception is if {es} or the Identity Provider is configured to force user to re-authenticate. This login scenario is called _Service Provider initiated login_.
-
 [float]
 ===== SAML and basic authentication
 
-SAML support in {kib} is designed to be the primary (or sole) authentication method for users of that {kib} instance. However, you can configure both SAML and Basic authentication for the same {kib} instance:
+SAML support in {kib} is designed to be the primary (or sole) authentication method for users of that {kib} instance. However, you can configure both SAML and Basic authentication for the same {kib} instance. This might be the case for {kib} or {es} admins whose accounts aren't linked to the Single Sign-On users database:
```

Comment:
> SAML support in {kib} is designed to be the primary (or sole) authentication method for users of that {kib} instance. 

*question* Is this still true now that we have the login selector screen? I was thinking that all configured providers could be treated equally now

Reply:
Yeah, I was thinking about this too. Let me describe the reason behind the decision to keep this sentence and you'll tell me if it's a reasonable justification or not. The main point is that we still don't/can't handle every use case on the same level as we do when we have a single provider: 

* user will still be able to log in with SAML even if selector _isn't_ enabled. In case of multiple providers it will depend on the `order`.
* and small things like when `overwritten_session` view is shown only if it's handled by the same SAML authentication provider

That's why I thought encouraging the simplest possible and well-supported scenario would make sense and advanced users will find their way in any case. But I don't really have a strong opinion on this and would appreciate your thoughts.

Reply:
Thanks for explaining! I personally think it'll be less confusing if we just omit this sentence altogether. I don't want to discourage folks from using multiple auth providers if it suites their needs, and I expect that most admins who require multiple auth providers are already in the "advanced users" category, so I'm not convinced this adds a lot of value

Reply:
Thanks for the input! Let's make it simple then, I'll remove this sentence for SAML and OpenID Connect.

Reply:
Done, for SAML, OpenID Connect and PKI.